src/test/ui/feature-gate-i128_type2.rs

```diff
@@ -10,20 +10,20 @@
 
 // gate-test-i128_type
```

Comment:
You'll probably have to remove this line.

Reply:
Thanks :) I removed the test altogether since there is already a test for the repr128 gate.